DESCRIPTION

```diff
@@ -1,15 +1,15 @@
 Package: dashCytoscape
-Title: A Component Library for Dash aimed at facilitating network visualization in Python, wrapped around Cytoscape.js
+Title: Interactive Network Visualization Component for Dash 
 Version: 0.1.1
 Authors @R: as.person(c(The Plotly Team <cytoscape@plot.ly>))
-Description: A Component Library for Dash aimed at facilitating network visualization in Python, wrapped around Cytoscape.js
+Description: A Component Library for Dash aimed at facilitating network visualization in R, wrapped around Cytoscape.js
 Depends: R (>= 3.0.2)
 Imports: dash
-Suggests: testthat, roxygen2
+Suggests: 
 License: MIT + file LICENSE
 URL: https://github.com/plotly/dash-cytoscape
 BugReports: https://github.com/plotly/dash-cytoscape/issues
 Encoding: UTF-8
 LazyData: true
 Author: The Plotly Team [aut]
-Maintainer: The Plotly Team <cytoscape@plot.ly>
+Maintainer: Ryan Patrick Kyle <ryan@plot.ly>
```

Comment:
Is there any reason that this was changed? I think it might be better to have the library emails instead of personal emails (just to remain consistent with everything).

Reply:
Yeah, unfortunately CRAN does not allow mailing lists for package submissions 🙁 

https://cran.r-project.org/web/packages/policies.html#Source-packages

> The package’s `DESCRIPTION` file must show both the name and email address of a single designated maintainer (a person, not a mailing list). That contact address must be kept up to date, and be usable for information mailed by the CRAN team without any form of filtering, confirmation …